l10n_jp_report_format/models/ir_actions_report.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Report(models.Model):
+    _inherit = "ir.actions.report"
+
+    commercial_partner = fields.Boolean()
```

Comment:
Please add a help text.
```suggestion
    show_commercial_partner = fields.Boolean(help="If selected, the commercial partner of the document partner will show in the report output (instead of the document partner).")
```